tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js

```diff
@@ -42,13 +53,22 @@ context('Collapse sidebar/appearance', () => {
                     expect(currentValueLeftBackground).to.be.greaterThan(defaultValueLeftBackground);
                 });
 
-            // wait when background fitted
-            cy.wait(500);
+            // Check issue 3250
+            cy.get('#cvat_canvas_content').invoke('attr', 'style').then((canvasContainerStyle) => {
+                cy.viewport(2999, 2999); // Resize window
+                cy.get('#cvat_canvas_content').should('have.attr', 'style').and('not.equal', canvasContainerStyle);
+                cy.viewport(1300, 960); // Return to the original size
```

Comment:
Can we get the original size before resizing and apply it after the test? In cypress.json, we can update viewport size, but probably we will not remember about this place. It is like a second place of the truth.

Reply:
Reworked to use values from cypress.json